readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -417,8 +417,8 @@ def test_get_invalid_raw_log(self):
 class APITests(TestCase):
     fixtures = ['eric.json', 'test_data.json']
 
-    def test_make_project(self):
-        """Test that a superuser can use the API."""
+    def test_cant_make_project(self):
+        """Test that a can't use the API to create projects."""
```

Comment:
We aren't using this in our code, I'm not sure if we allow this in the external API, we don't have that documented either https://docs.readthedocs.io/en/latest/api/v1.html